### PR TITLE
testing/amule: upgrade to 10929

### PR DESCRIPTION
**Pull request by AmatCoder**

### CODE REVIEW

**Comment by AmatCoder** [comment]
The Travis Ci build fails because "The log length has exceeded the limit of 4 MB"

**Comment by 7heo** [comment]
This is failing because of the build verbosity... Surely there's a way to make it less verbose (it's not even warnings it's mostly notes)

File: testing/amule/APKBUILD [hunk 1]
```diff
@@ -2,33 +2,25 @@
 # Maintainer: August Klein <amatcoder@gmail.com>
 pkgname=amule
 pkgsvn=aMule-SVN-r
```
Comment (jirutka) [inline testing/amule/APKBUILD]: This is not a standard abuild variable, so it must be prefixed with an underscore (`_pkgsvn`).
I would prefer to name it `_pkgname` instead, that's kinda a convention in abuilds.

**Comment by AmatCoder** [comment]
Fixed abuild variable and squashed.
Also I added `-Wno-unused-local-typedefs` to `$CPPFLAGS` to suppress most of warnings.

**Comment by jirutka** [comment]
Merged in ae3a0e238d0b088df86433f4e82243afaef835b4, thanks!

**Pull request closed without merging**